src/components/Quiz/Quiz.tsx

```diff
@@ -1,30 +1,38 @@
 import { useState } from 'react';
-
-import AnswerButton from './AnswerButton';
-import questions from './questions/webdev.json';
+import { AnswerButton } from './AnswerButton';
+import questionsJson from './questions/webdev.json';
 
 interface Question {
   question: string;
   options: string[];
   answer: string;
 }
 
-export default function Quiz() {
+const questions: Question[] = (questionsJson as unknown) as Question[];
```

Comment:
I don't get the type casting and assigning the type at the same time.
You can use the "satisfies" operator to make sure that the JSON file will match the interface type